src/makeRobot.js

```diff
@@ -38,7 +37,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name},`
+       + ` chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(distance = 1) {
+      if (distance < 0) {
+        return this;
+      }
+
+      this.coords.y += distance;
+
+      return this;
```

Comment:
```suggestion
      if (distance > 0) {
        this.coords.y += distance;
      }

      return this;
```
Consider refactoring the code to keep it simple and clean